lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -155,6 +155,35 @@ def test_contourf3d_fill():
     ax.set_zlim(-1, 1)
 
 
+@pytest.mark.parametrize('extend, levels', [['both', [2, 4, 6]],
+                                            ['min', [2, 4, 6, 8]],
+                                            ['max', [0, 2, 4, 6]]])
+@check_figures_equal(extensions=["png"])
+def test_contourf3d_extend(fig_test, fig_ref, extend, levels):
```

Comment:
Why is this parameterized over `extend` and `levels` if it they're not used for anything?

Reply:
Woops, should be fixed now to use these in generating the test image.